linchpin/cli/__init__.py

```diff
@@ -258,28 +259,39 @@ def _write_distilled_context(self, run_data):
             except Exception:
                 pass
```

Comment:
I should have mentioned before that masking exceptions is really bad practice.  Can you fix this?

Reply:
In general, I would agree with you. In this case, if the context distiller fails to collect the data, the exception is unimportant. I didn't want to have the code fail, or have to rethrow an exception. Instead, if an exception occurs, the distiller will record what it has (sometimes no data), and write it out to the linchpin.distilled file.  This would usually only happen on an incomplete provision, or when someone is modifying the fields which are being pulled into the distiller.